README.md

```diff
@@ -285,12 +286,8 @@ signIn.renderEl(
   },
 
   function error(err) {
-    // The widget will handle most types of errors - for example, if the user
-    // enters an invalid password or there are issues authenticating.
-    //
     // This function is invoked with errors the widget cannot recover from:
-    // 1. Known errors: CONFIG_ERROR, UNSUPPORTED_BROWSER_ERROR, OAUTH_ERROR
-    // 2. Uncaught exceptions
+    // Known errors: CONFIG_ERROR, UNSUPPORTED_BROWSER_ERROR, OAUTH_ERROR, REGISTRATION_FAILED
```

Comment:
@nikhilvenkatraman-okta 
Would you add more colors to why we invoke globalError handler when registration failed?
Given this is not doc'ed, I wonder if it's safe to refactor to use `afterError` event?

Reply:
@haishengwu-okta `REGISTRATION_FAILED` is invoked when the regCallbacks fails. We have `parseSchema `,`preSubmit ` and `postSubmit` hooks for registration. We invoke the global error because at this point the user registration fails and they cant proceed further. Looking at the readme for `afterError ` event, I don't see a problem with refactoring the regCallbacks to use `afterError ` event.